ios/src/RCTACPMedia.m

```diff
@@ -34,33 +34,31 @@ - (dispatch_queue_t)methodQueue
         _trackers = [NSMutableDictionary dictionary];
     }
 
-    [ACPMedia createTracker:^(ACPMediaTracker * _Nullable tracker) {
-        if (!tracker) {
-            reject(@"createTracker failed", @"Tracker was nil", nil);
-            return;
-        }
-                   
-        NSString *uuid = [[NSUUID UUID] UUIDString];
-        self->_trackers[uuid] = tracker;
-        resolve(uuid);
-    }];
+    ACPMediaTracker* tracker = [ACPMedia createTracker];
+    if (!tracker) {
+        reject(@"createTracker failed", @"Tracker was nil", nil);
+        return;
+    }
+
+    NSString *uuid = [[NSUUID UUID] UUIDString];
+    self->_trackers[uuid] = tracker;
```

Comment:
Now that we are not within a completion block can we just do `_trackers[uuid] = tracker;`?